waiter/specs/k8s-default-pod.edn

```diff
@@ -0,0 +1,38 @@
+{:kind "ReplicaSet"
+ :apiVersion "extensions/v1beta1"
```

Comment:
Should this come from the Clojure code as many of the api queries use this version string too? Maybe it needs to be a constant in the kubernetes scheduler code?